core/trino-main/src/main/java/io/trino/sql/planner/LocalExecutionPlanner.java

```diff
@@ -490,7 +489,8 @@ public LocalExecutionPlan plan(
                 outputLayout,
                 types,
                 partitionedSourceOrder,
-                new PartitionedOutputFactory(
```

Comment:
there are more usages of `PartitionedOutputFactory` (also in tests). Use operator factories there too